cmd/podman/logs.go

```diff
@@ -54,6 +54,7 @@ func init() {
 	flags.StringVar(&logsCommand.Since, "since", "", "Show logs since TIMESTAMP")
 	flags.Int64Var(&logsCommand.Tail, "tail", -1, "Output the specified number of LINES at the end of the logs.  Defaults to -1, which prints all lines")
 	flags.BoolVarP(&logsCommand.Timestamps, "timestamps", "t", false, "Output the timestamps in the log")
+	flags.BoolVarP(&logsCommand.Names, "Names", "n", false, "Show the container name in the log")
```

Comment:
Should be name not Names.
Lower case.  

Variable should be something like logsCommand.UseName, so that it would be easier to read below.  When I see the variable Name, I expect it to be a string.